### PR TITLE
Tool for visualizing the wiki

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: dev/wiki-visualizer/bin/wiki_visualizer.dart [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  final WikiPage sidebar = wiki.pageForTitle('_Sidebar');
+  final Set<WikiPage> accessibleContent = sidebar.findAllAccessible();
+
+  print('strict digraph {');
+  final StringBuffer attributes = StringBuffer();
+  for (final WikiPage page in wiki.pages) {
+    // if (page == sidebar) {
+    //   continue;
+    // }
```
Comment (keyonghan) [inline dev/wiki-visualizer/bin/wiki_visualizer.dart:33]: Nit: remove?

File: dev/wiki-visualizer/bin/wiki_visualizer.dart [hunk 2]
```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:wiki_visualizer/wiki_page.dart';
+import 'package:wiki_visualizer/dot.dart';
```
Comment (keyonghan) [inline dev/wiki-visualizer/bin/wiki_visualizer.dart:8]: Nit: order alphabetically.

**Review by keyonghan — APPROVED** [review]
LGTM with nits.

File: dev/wiki-visualizer/bin/wiki_visualizer.dart (hunk 1 above)
Comment (Hixie) [reply dev/wiki-visualizer/bin/wiki_visualizer.dart:33]: done

File: dev/wiki-visualizer/bin/wiki_visualizer.dart (hunk 2 above)
Comment (Hixie) [reply dev/wiki-visualizer/bin/wiki_visualizer.dart:8]: weird, why didn't that get caught by the analyzer.... oh, i am probably not analyzing any of this code! let me update the CI stuff and try again.

Comment (Hixie) [reply dev/wiki-visualizer/bin/wiki_visualizer.dart]: ok added this and the other one in dev/ to `tests.yaml`

**Pull request merged**